### PR TITLE
chore (logging): Added logging to debug activity alerts feature in production environment.

### PR DESCRIPTION
**Pull request by saleem-latif**
This PR adds logging in the activity alerts feature, this will help us in debugging the cause of the problem with activity alerts feature in production plugin version.

### CODE REVIEW

File: src/main/java/org/intellij/sdk/codesync/CodeSyncLogger.java [hunk 1]
```diff
@@ -16,11 +16,21 @@
...
 
 public class CodeSyncLogger {
-    private final static Logger LOGGER = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+    public static void logConsoleMessage(String message) {
+        logConsoleMessage(message, LogMessageType.INFO);
+    }
+
+    /*
+    This will log given message to the console. It will not send the message to cloud watch.
+    This is useful only with local debugging or special cases when we can get log file from the client.
```
Comment (Basit-qc) [inline src/main/java/org/intellij/sdk/codesync/CodeSyncLogger.java:28]: 👍

File: src/main/java/org/intellij/sdk/codesync/HandleBuffer.java [hunk 2]
```diff
@@ -122,7 +122,7 @@ public void run() {
                 try {
                     HandleBuffer.handleBuffer(project);
                 } catch (Exception e) {
-                    System.out.println("handleBuffer exited with error:");
+                    CodeSyncLogger.error(String.format("handleBuffer exited with error: %s", e.getMessage()));
```
Comment (Basit-qc) [inline src/main/java/org/intellij/sdk/codesync/HandleBuffer.java:125]: Good call on this

File: src/main/java/org/intellij/sdk/codesync/Utils.java [hunk 3]
```diff
@@ -294,8 +294,6 @@ public static void handleDocumentUpdates(VirtualFile file, String repoPath, Stri
             CodeSyncLogger.error("Skipping the update event, file is null.");
             return;
         }
-        float time = System.currentTimeMillis();
-        System.out.println(String.format("Event: %s", time));
```
Comment (Basit-qc) [inline src/main/java/org/intellij/sdk/codesync/Utils.java:298]: 👍 Line added long ago :D

File: src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java [hunk 4]
```diff
@@ -175,7 +175,7 @@ public Integer uploadFile(String accessToken, ConfigRepo configRepo, DiffFile di
                 this.uploadToS3(originalsFile, preSignedURLData);
             }
         } catch (ClassCastException error) {
-            System.out.println("Could not upload the file.");
+            CodeSyncLogger.logConsoleMessage("Could not upload the file.");
             // this would probably mean that `url` is empty, and we can skip aws upload.
```
Comment (Basit-qc) [inline src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java:179]: Not part of this PR though, If "url" is empty, I think it shouldn't go to s3

**Review by Basit-qc — APPROVED** [review]

**Review by syedgulahmed — APPROVED** [review]
Good work!!!
